keep-ui/utils/fatigue.ts

```diff
@@ -18,7 +18,7 @@ export const calculateFatigue = (
   }
 
   // Sort alerts by timestamp
-  const sortedAlerts = alerts.sort(
+  const sortedAlerts = [...alerts].sort(
```

Comment:
If you don't want to copy arrays when sorting, you can use [`toSorted()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)